rhino-deinst

```diff
@@ -6,7 +6,11 @@ if ! command -v dialog > /dev/null; then
   sudo apt-get install dialog -y
 fi
 
-SELECTED=($( dialog --backtitle "Press 'space' to toggle an option, use arrow keys to go up/down, press enter to continue" --scrollbar --separate-output --checklist "Select Desktop Environment(s)" $(expr $(tput lines) - 5) $(expr $(tput cols) - 5) 5 \
+# Set LINES and COLUMNS variables. (Reference: https://stackoverflow.com/a/48016366)
+shopt -s checkwinsize
+cat /dev/null
```

Comment:
what's this for?

Reply:
To not use stty and command substitution to get the terminal's length and columns.